src/main/java/com/app/library/model/entity/BookCategoryEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.app.library.model.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.app.library.constant.ApplicationConstants.DB_SCHEMA;
+
+@Entity
+@Table(name = "book_category", schema = DB_SCHEMA)
+@Data
```

Comment:
As we discussed, we dont need @Data until we don't need @EqualsAndHashCode and @ToString annotations :) Please use only needed annotations